_posts/2020-10-26-dynamically-changeable-logging.md

```diff
@@ -0,0 +1,150 @@
+---
+layout: post
+title:  "Dynamically changeable logging levels"
+date: 2020-10-26
+author: stanislav_knot
+---
+
+Logging is an indisputably important feature for applications.
+Logging monitors what the application is doing, but equally importantly,  it logs what the application is not doing. 
+Looking into application logs can help us prevent an application remaining or even getting into a faulty state.
```

Comment:
Can you add here one or two sentences about why is it important to change the log levels dynamically? No restarts, you can fine-tune it to catch the right problem etc?

Reply:
It is mentioned in the conclusion.